app/routes/billing.js

```diff
@@ -0,0 +1,26 @@
+import Route from '@ember/routing/route';
+import {inject as service} from '@ember/service';
+
+export default Route.extend({
+    billing: service(),
+
+    queryParams: {
+        action: {refreshModel: true}
+    },
+
+    init() {
+        this._super(...arguments);
+    },
```

Comment:
Not needed as you have no custom logic, without the override the base class' `init()` will be called anyway